community/index.md

```diff
@@ -53,19 +53,21 @@ to begin <img src="{{ site.baseurl }}assets/img/icon_smile.gif" alt=":-)">
 
 mySociety has very limited capacity to run a handful of Alaveteli sites for willing volunteers. Contact our <a href="mailto:international@mysociety.org">international team</a> for more information.
 
-## IRC
+## Slack
+
+### alaveteli.slack.com
 
-### irc.mysociety.org #mschat
+mySociety developers and re-users can be found on the Alaveteli Slack, and
+unlike on IRC there's a handy history feature built in so that conversations can
+more easily take place across time zones (hello again Australia, we missed you!)
+<a href="email clientsupport@alaveteli.org mailto:clientsupport@alaveteli.org">Contact us for an invite</a>.
 
-**`#mschat`** &mdash;
-the mySociety developers can be found on IRC in `#mschat` on
-`irc.mysociety.org` (running on port `6667`, and also `9219`)
-(also available via this [web interface](http://www.irc.mysociety.org)).
+## IRC
 
 ### irc.freenode.net #alaveteli
 
 **`#alaveteli`** &mdash;
-there's an `#alaveteli` channel on
+If you prefer not to use Slack, there's an `#alaveteli` channel on
```

Comment:
We were going to direct them to `freenode#mysociety`, rather than maintain a separate channel.